art/Framework/Services/Optional/MemoryTrackerLinux_service.cc

```diff
@@ -425,17 +426,35 @@ namespace art {
     mf::LogAbsolute log{"MemoryTracker"};
     HorizontalRule const rule{100};
     log << '\n' << rule('=') << '\n';
-    log << std::left << "MemoryTracker summary (base-10 MB units used)\n\n";
-    log << "  Peak virtual memory usage (VmPeak)  : " << unique_value(rVMax)
-        << " MB\n"
-        << "  Peak resident set size usage (VmHWM): " << unique_value(rRMax)
-        << " MB\n";
-    if (!(fileName_.empty() || fileName_ == ":memory:")) {
-      log << "  Details saved in: '" << fileName_ << "'\n";
+
+    if (anyTableFull_()) {
+      log << "The SQLite database connected to the MemoryTracker exceeded the "
+             "available resources.\n";
+      log << "No memory usage summary is available.\n";
+      log << "The database at " << fileName_
+          << " will contain an incomplete record of this job's memory usage.\n";
```

Comment:
@marcpaterno, can this be guarded with with the same type of check as at line 442 below?